src/pages/library/DataBrowserPreview.js

```diff
@@ -19,8 +19,20 @@ const styles = {
   table: {
     header: {
       color: colors.accent(),
-      height: '1rem',
+      height: '2rem', lineHeight: '2rem',
       textTransform: 'uppercase', fontWeight: 600, fontSize: '0.75rem'
+    },
+    headerRowStyle: {
+      borderTop: '1px solid rgba(0,0,0,0.3)', borderBottom: '1px solid rgba(0,0,0,0.3)'
```

Comment:
I lied when I said that was my last comment; could we change all these to use `colors.dark`?  Reason being that we don't actually use transparency there, we calculate a similar effect, so that there isn't a render cost or change based on background color.